index.html

```diff
@@ -23,32 +23,46 @@
       rel="stylesheet"
       href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
     />
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+   
   </head>
 
   <body>
     <header>
-      <nav class="mobile-version">
+      <nav class="mobile-version" id="mobile-nav">
         <div class="menu">
           <div class="shape">
             <div id="triangle"></div>
             <div id="diamond"></div>
           </div>
           <div>
-            <img id="menuimg" src="./assets/img/menu.png" alt="square" />
+            <button class="hamburger" id="menuimg" >
+              <i class="fa-sharp fa-solid fa-bars  fa-3x " id="hamburgericon"></i>
+            </button>
           </div>
         </div>
       </nav>
-      <nav class="desktop-version">
+      <div id="myNav" class="overlay">
+        <a href="javascript:void(0)" class="closeicon" ><i class="closeIcon material-icons  fa-3x" id="close-icon">close</i></a>
+        <div class="overlay-content">
+          <a href="#intro" onclick="closeNav()">Hello</a>
+          <a href="#projects" onclick="closeNav()">Portfolio</a>
+          <a href="#about" onclick="closeNav()">About</a>
+          <a href="#form" onclick="closeNav()">Contact</a>
+        </div>
+      </div>
```

Comment:
You did a great job working on your mobile menu! 🚀 In fact, all the functionality is working as expected, but according to the requirements, you should stick as much as possible to the Figma design 🎨.

- [ ] Please consider using the correct background image for the menu. 🚧 
- [ ] Please consider using the font provided by the template as well as the correct color. 🚧 

|Template|Your design|
|---|---|
| ![image](https://user-images.githubusercontent.com/77069011/208725184-b0a96b10-1ca4-42d6-b9ac-c33215656019.png) | ![image](https://user-images.githubusercontent.com/77069011/208725045-acb7ae2e-fec4-454d-98b9-5cf198c7bf97.png) |

Having a good-looking and consistent design will show future recruiters what you're capable of and also catch their attention! So please consider investing some time in applying these changes. 🤓 🛠️